packages/evaluation/src/mcpclient/mobileWebMcpClient.ts

```diff
@@ -9,6 +9,8 @@ import { Client } from '@modelcontextprotocol/sdk/client/index.js';
 import { StdioClientTransport } from '@modelcontextprotocol/sdk/client/stdio.js';
 import { fileURLToPath } from 'url';
 import { dirname, join } from 'path';
+import { spawn, ChildProcess } from 'child_process';
+import path from 'path';
```

Comment:
Remove this line and import `resolve` in line 11. Fix line 76.

Reply:
done